Health.py

```diff
@@ -1,39 +1,78 @@
-# For Healing:
-# vitC heals health
-# vitB heal magic
-# supplies repair mechanical
+FoodDict = {"Bread": [10, 0],
+            "Apple": [5, 0],
+            "Chicken Head": [20, 20],
+            "Thorny Fruit": [-20, -20]
+            }
 
-# food item would be ['foodName', 'food', [vitC_Value, vitB_Value]]
+RepairItems = {"Wrench": [10],
+               "Toolkit": [30],
+               "Repair Box": [100]
+            }
 
-# Enemy has health/magic/mechanical meters, attack & defense profile, Loot
-# If mechanical or health are zero, you cannot attack
-# If magic is zero, you may not use attacks with more than 5 magic damage
-# Defense profile is additive and subtracts from attack
-# Only one attack at a time, alternating turns
+PlayerStats = {"Health": 100,
+             "Magic": 100,
+             "Supply": 100}
```

Comment:
What is supply?

Reply:
Here was my outline for my function: 

# For Healing:
# vitC heals health
# vitB heald magic
# supplies repair mechanical

# Enemy has health/magic/mechanical meters, attack & defense profile, Loot 
# If mechanical or health are zero, you cannot attack
# If magic is zero, you may not use attacks with more than 5 magic damage
# Defense profile is additive and subtracts from attack
# Only one attack at a time, alternating turns


Supply in this case is just the "Mechanical" meter, which I should probably rename it to that.

Reply:
Sorry I don't know how to edit the font size in this editor lol

Reply:
lol. Thanks for clarifying.